guides/source/active_job_basics.md

```diff
@@ -388,6 +389,24 @@ class GuestsCleanupJob < ApplicationJob
 end
 ```
 
+As described in https://guides.rubyonrails.org/active_record_callbacks.html#halting-execution,
```

Comment:
I don’t think it’s appropriate to link to docs for Active Record. The underlying callback mechanism is [`ActiveSupport::Callbacks`](https://api.rubyonrails.org/classes/ActiveSupport/Callbacks/ClassMethods.html#method-i-define_callbacks).

Reply:
Sadly those docs don't succinctly describe the callback halting mechanism like the AR docs do. 

However, that's what this PR change should achieve so I'm happy to remove it.

Reply:
updated to add a link to https://api.rubyonrails.org/classes/ActiveSupport/Callbacks.html which outlines the halting mechanism and has a link to detailed implementation that you posted above. 
> By default callbacks are halted by throwing :abort. See [ClassMethods#define_callbacks](https://api.rubyonrails.org/classes/ActiveSupport/Callbacks/ClassMethods.html#method-i-define_callbacks) for details.

Reply:
> Sadly those docs don't succinctly describe the callback halting mechanism like the AR docs do.

I'll gladly accept a PR to improve that documentation a bit.
- The `Record` example can be move up a bit, after the second paragraph?
- Another code example could be added to demonstrate `throw :abort` after the third paragraph ("By default callbacks are halted by ..."
- The fourth paragraph: "Three kinds of callbacks are supported:" could probably use some better formatting. Maybe as a list?